examples/fact-check-extension/scripts/postbuild.js

```
@@ -0,0 +1,87 @@
+  const manifestPath = path.join(distDir, 'manifest.json');
+
+  if (!fs.existsSync(manifestPath)) {
+    console.error('manifest.json not found in dist directory');
+    return;
+  }
+
+  try {
+    const manifest = JSON.parse(fs.readFileSync(manifestPath, 'utf8'));
+
+    // Fix background script path
+    if (manifest.background && manifest.background.service_worker) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)